ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Tables.hs

```diff
@@ -0,0 +1,553 @@
+{-# LANGUAGE ConstraintKinds    #-}
+{-# LANGUAGE DataKinds          #-}
+{-# LANGUAGE DerivingVia        #-}
+{-# LANGUAGE FlexibleContexts   #-}
+{-# LANGUAGE FlexibleInstances  #-}
+{-# LANGUAGE GADTs              #-}
+{-# LANGUAGE LambdaCase         #-}
+{-# LANGUAGE PolyKinds          #-}
+{-# LANGUAGE Rank2Types         #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TypeFamilies       #-}
+-- |
+
+module Ouroboros.Consensus.Ledger.Tables (
```

Comment:
Since we're creating this module, I think it's a great opportunity for writing proper documentation for this module. The concept of ledger tables is far from trivial. We could make this in a follow up PR of course.

Reply:
Done